spec/models/king_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe King, type: :model do
+	describe "is_valid? method" do
+    it "should check if king can move 1 space up" do
+    	user = FactoryBot.create(:user)

```

Comment:
You could follow the readme [here](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#configure-your-test-suite) so there will be no need to write `create(:user)` instead of `FactoryBot...` :)